Documentation/html_viewer/js/dictionary_functions.js

```diff
@@ -108,19 +112,35 @@
 			return tMatched;
 		});
 
-		// Sort the priority matches to the top
-		tState . searched . data . sort(function(a, b) 
-		{
-			var tToMatch = collectSyntax(a);		
-			if (priorityRegex.test(tToMatch))
-				return -1;
-			
-			tToMatch = collectSyntax(b);
-			if (priorityRegex.test(tToMatch))
-				return 1;
-				
-			return 0;
-		});
+        // Sort the priority matches to the top
+        tState . searched . data . sort(function(a, b) 
+        {
+            var tNameA = a["display name"].toLowerCase();
+            var tNameB = b["display name"].toLowerCase();
+
+            // full word matches in display name sort to the top
+            var tWordA = wordRegex.test(tNameA);
+            var tWordB = wordRegex.test(tNameB);
+            if (tWordA && !tWordB)
+                return -1;
+            if (tWordB && !tWordA)
+                return 1;
+
+            // display names with words that start with terms prioritized
+            var tPriorityA = priorityRegex.test(tNameA);
+            var tPriorityB = priorityRegex.test(tNameB);
+            if (tPriorityA && !tPriorityB)
+                return -1;
+            if (tPriorityB && !tPriorityA)
+                return 1;
+
```

Comment:
This is where the addition would be made to include 3. above.  I would probably want to change to using tMatchA & tMatchB for each of the 3 sets of Regex.  The third would use regex.test(tName?).  This will slow things down some, but I have not had a chance to build a page with timers to check all of the performance variations.

Reply:
@bwmilby The alphabetical sort would mean that wordRegex isn't necessary wouldn't it or is it faster to return that here?

Reply:
wordRegex will ensure an exact match that isn't the first word appears at the top of the list.

Reply:
The wordRegex groups whole word matches at the top of the list so "pane" will have the 2 entries that match ahead of the ones with "panel" for example.  Without wordRegex, they are not separated and are just sorted alphabetically.

I'm trying to envision where adding the extra regex will help find things.  "the" is an example where it would change the order, but I'm not sure if it is helpful.  After "there" matches, you either get "dither" or "abs" as your next match.  Adding the extra regex will put matches in the middle/end of a word in the display name ahead of ones that have that kind of match only in the syntax.  I have the change stashed and can submit.

Reply:
sorry... I was being stupid and thinking the word and priority matches were for the beginning of the display name not anywhere in it